python/BioSimSpace/Protocol/_AToM.py

```diff
@@ -0,0 +1,3287 @@
+from BioSimSpace.Units.Area import angstrom2
```

Comment:
Could this be called `_atm.py` to match the modules in the other sub-modules, e.g. `Process._atm` etc.

Reply:
This is now updated (https://github.com/OpenBioSim/biosimspace/blob/feature_AToM/python/BioSimSpace/Protocol/_atm.py)